tests/step-039.sh

```diff
@@ -12,6 +12,14 @@ mkdir -p /tmp/start/dir && cd /tmp/start/dir || exit 1
 mkdir -p /tmp/empty &&
     GIT_TEMPLATE_DIR=/tmp/empty git init || exit 1
 
+git config --local githooks.single.install "Y"
+OUT=$(sh /var/lib/githooks/install.sh --single 2>&1)
+if ! echo "$OUT" | grep -iq "DEPRECATION WARNING: Single install"; then
+    echo "! Expected installation to fail because of single install flag: $OUT"
+    exit 1
+fi
+git config --local --unset githooks.single.install
+
 if ! sh /var/lib/githooks/install.sh --single; then
```

Comment:
This should also fail, regardless of the `githooks.single.install` setting.
Once this PR is in, you can only update (e.g. run the install) if it's not single anymore.